pallets/invulnerables/src/lib.rs

```diff
@@ -213,4 +263,31 @@ pub mod pallet {
             Ok(())
         }
     }
+
+    use pallet_session::SessionManager;
+    use sp_staking::SessionIndex;
```

Comment:
Move these imports to the top of the file

Reply:
done